pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -170,19 +174,38 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 		return err
 	}
 
-	if _, err := pna.kubeClient.CoreV1().Services(pna.namespaceName).Create(context.Background(), podNetworkTargetService, metav1.CreateOptions{}); err != nil {
+	service, err := pna.kubeClient.CoreV1().Services(pna.namespaceName).Create(context.Background(), podNetworkTargetService, metav1.CreateOptions{})
+	if err != nil {
 		return err
 	}
 
 	hostNetworkTargetDeployment.Spec.Replicas = &numNodes
 	if _, err := pna.kubeClient.AppsV1().Deployments(pna.namespaceName).Create(context.Background(), hostNetworkTargetDeployment, metav1.CreateOptions{}); err != nil {
 		return err
 	}
-
 	if _, err := pna.kubeClient.CoreV1().Services(pna.namespaceName).Create(context.Background(), hostNetworkTargetService, metav1.CreateOptions{}); err != nil {
 		return err
 	}
 
+	serviceEnv := []corev1.EnvVar{
+		{
+			Name:  "SERVICE_CLUSTER_IP",
+			Value: service.Spec.ClusterIP,
+		},
+	}
+	podNetworkServicePollerDep.Spec.Replicas = &numNodes
+	podNetworkServicePollerDep.Spec.Template.Spec.Containers[0].Image = openshiftTestsImagePullSpec
+	podNetworkServicePollerDep.Spec.Template.Spec.Containers[0].Env = append(podNetworkServicePollerDep.Spec.Template.Spec.Containers[0].Env, serviceEnv)
```

Comment:
how does this append avoid having two env vars with the same name since the manifests still have the SERVICE_CLUSTER_IP?

Reply:
yep that won't help, changed it again.